core-scroll-threshold.html

```diff
@@ -125,14 +136,22 @@
       this._boundScrollHandler = this.checkThreshold.bind(this);
     },
 
+    detached: function() {
+      // Remove listener for any previous scroll target
+      if (this._scrollTarget) {
+        this._scrollTarget.removeEventListener(this._boundScrollHandler);
+      }
+    },
+
     setup: function() {
+      var target = this.scrollTarget || this;
+
       // Remove listener for any previous scroll target
-      if (this._scrollTarget && (this._scrollTarget != this.target)) {
+      if (this._scrollTarget && (this._scrollTarget != target)) {
         this._scrollTarget.removeEventListener(this._boundScrollHandler);
```

Comment:
`this._scrollTarget.removeEventListener('scroll', this._boundScrollHandler);`
?